README.md

```diff
@@ -197,6 +200,12 @@ The choice is defined with the variable `kube_network_plugin`. There is also an
 option to leverage built-in cloud provider networking instead.
 See also [Network checker](docs/netcheck.md).
 
+## Ingress Plugins
+
+- [ambassador](docs/ambassador.md): the Ambassador Ingress Controller and API gateway.
+
+- nginx: the NGINX Ingress Controller
```

Comment:
Maybe add a link to https://github.com/kubernetes/ingress-nginx ?

Reply:
oops. sorry. my link https://kubernetes.github.io/ingress-nginx/ looks cooler ;)